alkira/resource_alkira_policy_nat_rule.go

```diff
@@ -151,7 +151,23 @@ func resourceAlkiraPolicyNatRule() *schema.Resource {
 								"invalidate. Default is `true`.",
 							Type:     schema.TypeBool,
 							Optional: true,
-							Default:  true,
+						},
+						"src_addr_translation_routing_track_prefixes": {
+							Description: "The list of prefixes to track.",
+							Type:        schema.TypeList,
+							Elem:        &schema.Schema{Type: schema.TypeString},
+							Optional:    true,
+						},
+						"src_addr_translation_routing_track_prefix_list_ids": {
+							Description: "The list of prefix list IDs.",
+							Type:        schema.TypeList,
+							Elem:        &schema.Schema{Type: schema.TypeInt},
+							Optional:    true,
+						},
+						"src_addr_translation_routing_invalidate_track_prefixes": {
```

Comment:
If we decided to call those fields with prefix `src_addr_translation_routing_track_`, then let's keep them consistent.